### PR TITLE
build for ppc64le

### PR DESCRIPTION
**Pull request by synapticarbors**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by synapticarbors** [comment]
@conda-forge-admin please rerender

**Comment by synapticarbors** [comment]
@conda-forge/help-ppc64le Any idea why the two failure here are occurring for travis?

```
subprocess.CalledProcessError: Command '['git', 'clone', '--depth=1', 'https://github.com/conda-forge/feedstock-outputs.git', '/tmp/tmpnl6uudq7_recipe/feedstock-outputs']' returned non-zero exit status 128.
```

**Comment by jayfurmanek** [comment]
```
fatal: unable to access 'https://github.com/conda-forge/feedstock-outputs.git/': Could not resolve host: github.com
```

Looks like a DNS hiccup? Has this happened multiple times in a row? (tried a restart on it?)

**Comment by synapticarbors** [comment]
@jayfurmanek what's the recommended way of restarting it?

**Comment by isuruf** [comment]
You should see a Re-run button at https://github.com/conda-forge/blessings-feedstock/pull/8/checks?check_run_id=728296576

**Comment by synapticarbors** [comment]
I hit the "re-run" button for Travis and it's just been sitting there pulsing the "Re-running checks..." with the little github octocat logo for 2 hours

**Comment by isuruf** [comment]
I restarted from travis-ci.com ui.

**Comment by synapticarbors** [comment]
Thanks @isuruf. 

It looks like one of the two now passes, but one is still failing with the same error

**Comment by isuruf** [comment]
cc @beckermr

File: conda-forge.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 appveyor:
   secure: {BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1}
+provider:
+  linux_ppc64le: default
```
Comment (isuruf) [inline conda-forge.yml:4]: Can you add `linux_aarch64` as well?

**Comment by synapticarbors** [comment]
@conda-forge-admin please rerender

File: conda-forge.yml (hunk 1 above)
Comment (synapticarbors) [reply conda-forge.yml:4]: I added that. Is there a reason that not adding `linux_aarch64` could be an issue? Or is it just that conda-forge would prefer to have both available?

**Comment by beckermr** [comment]
Why did you cc me @isuruf? I am not following.

**Comment by isuruf** [comment]
Git cloning `feedstock-outputs` fails. I thought that we weren't using git operations anymore.

**Comment by beckermr** [comment]
That is just in the admin web service. We can remove them here too, but this shouldn't ever fail and doesn't need to be performant. The internet will fail at random from time to time.

**Pull request merged**